pyproject.toml

```diff
@@ -18,6 +20,7 @@ flake8 = "^3.6"
 flake8-isort = "^2.6"
 pylint = "^2.2"
 mypy = "^0.660.0"
+requests-mock = "^1.5"
```

Comment:
all dependencies are exactly what I would expect for Python to integrate with the REST API of Elasticsearch